tests/Base/QueryDataReaderTest.php

```diff
@@ -168,23 +166,9 @@ public function testDtoCreateItem(): void
         }
     }
 
-    public function testObjectCreateItem(): void
```

Comment:
Why it was deleted?

Reply:
`CustomerDataReader` in other tests works with object also.